proto/kvrpcpb.proto

```diff
@@ -766,6 +766,13 @@ enum IsolationLevel {
     RC = 1; // RC = read committed
 }
 
+// Operation allowed info during each TiKV storage threshold.
+enum DiskFullOpt {
+    AllowedOnNormal = 0; // the default value, means all operations are allowed when disk status is normal.
```

Comment:
```suggestion
    Normal = 0; // the default value, means all operations are allowed when disk status is normal.
```

Reply:
CommandPri::Normal

Reply:
I prefer `Normal` too. I think the compile error is not related to this change.

Reply:
I know why `Normal` is bad. So please change to `Default`.

Reply:
ok, done!